intro.tex

```diff
@@ -12,5 +12,5 @@ \section{INTRODUCTION}
 Evolution, not only of the code base but also of the development process itself, was embraced; developers were empowered to make changes.
 Given the distributed team, it is important that communication channels are open and easy to use and that our tools evolve as community standards evolve.
 Being agile enough to be able to migrate from one tool to another during the lifetime of a project is key when the software, processes, and people, change over what will be a 25 year period once the 10-year survey completes.
-For example, over the years we have migrated the codebase from Subversion to git (\S\ref{sec:subversion}); we have switched instant messaging from HipChat to Slack (\S\ref{sec:slack}); we have migrated continuous integration from Buildbot on our own hosts to Jenkins running in the cloud (\S\ref{sec:jenkins}); and we have moved documentation standards from Doxygen to Sphinx (\S\ref{sec:sphinx_tooling}).
+For example, over the years we have migrated the codebase from Subversion to git (\S\ref{sec:subversion}); we have switched instant messaging from HipChat to Slack (\S\ref{sec:slack}); we have migrated continuous integration from Buildbot on our own hosts to Jenkins running in the cloud (\S\ref{sec:jenkins}); and we have extended our documentation standards from just Doxygen to include Sphinx (\S\ref{sec:sphinx_tooling}) as well.
```

Comment:
Doxygen is still in use and we intend it to coexist with Sphinx in the long-term (via Breathe).